volume/drivers/coprhd/coprhd.go

```diff
@@ -0,0 +1,262 @@
+package coprhd
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/portworx/kvdb"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/volume"
+
+	napping "gopkg.in/jmcvetta/napping.v3"
```

Comment:
The goimports tool removes unused imports, since this package has a unconventional name it gets removed without an explicit name.